sagemathinc/cocalc
# Ts project settings.cjsx

# Description
Typescript `project_settings.cjsx` into `project/settings/`

# Testing Steps
1. Create a project
1. Add upgrades
1. Change the title and the description
1. Add a collaborator
1. Remove a collaborator
1. Stop and restart the project
1. Hide and delete the project
1. Check that features and configuration shows checkboxes
1. Start a Jupyter lab and hub server
1. Restart sagews

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):

Front end:
